episodes/25-wrap-up.md

```diff
@@ -47,12 +47,18 @@ This exercise should take 5 minutes.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+::: instructor
+
+You will need to get the post-training survey link yourself and provide it in the activity below. It should follow the form `https://carpentries.typeform.com/to/cjJ9UP#slug=...../` and can be found in an email trainers receive from the Carpentries team and on the website for the specific training.
```

Comment:
I think per #1732 we are not doing the individual workshop websites anymore, @sheraaronhurt or @tobyhodges can you confirm? 

```suggestion
You will need to get the post-training survey link yourself and provide it in the activity below. It should follow the form `https://carpentries.typeform.com/to/cjJ9UP#slug=...../` and can be found in an email trainers receive from the Carpentries Core Team before the training.
```

Reply:
@sheraaronhurt or @tobyhodges can we have confirmation on this?

Reply:
This looks right to me. Indeed Trainers are not required to create websites for individual events anymore.